### PR TITLE
Cross post of "Creando tu r-universe"

### PR DESCRIPTION
**Pull request by steffilazerte**
Original from: https://yabellini.netlify.app/es/post/r_universe/

Author check-list

* [X] I have read the Content Guidelines.
* [X] I have read the Technical Guidelines.
* [X] I used or followed the R Markdown or Markdown template.
* [X] I have followed the Style Guide.
* [X] I created or updated my author metadata with correct folder name.
* [X] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [X] I have added the "tech notes" tag if this is a technote.
* [X] I ran `roblog::ro_lint_md()` on index.md (optional).
* [X] I ran `roblog::ro_check_urls()` on index.md (optional).

Editor check-list

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [x] alternative text of images is informative
* [x] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

### CODE REVIEW

**Comment by steffilazerte** [comment]
Okay! @yabellini it's ready for your review! Could you 

- take a look and make sure I haven't missed anything?
- let me know what ALT text I can use for the pictures (in Spanish)
- and let me know if it's okay that I change "d" to "c" in the "Paso 2. detallar los paquetes que agregaremos a nuestro universo" section

@stefaniebutland I put down Tuesday Sept 21st as the publication date, okay?

@yabellini I *think* you should be able to just add those details here as a comment. If you can take a look by Monday (Sept 20th), that'd be ideal :) Other wise we'll just push back the publication date, no problem!

@stefaniebutland and @yabellini, are you both satisfied with the text at the top, linking to Yanina's original post?

**Comment by stefaniebutland** [comment]
Looks great to me. Thanks @steffilazerte!

> Tuesday Sept 21st as the publication date, okay?

yes!

**Comment by stefaniebutland** [comment]
@yabellini also check your rOpenSci author page [preview](https://deploy-preview-246--ropensci.netlify.app/author/yanina-bellini-saibene/) 🙂 and request any changes.

It is linked from the blog post and will appear at https://ropensci.org/author/ when published

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+### Paso 3: instalar la aplicación R-universe en GitHub
+
+En este paso debes instalar la aplicación R-universe en tu cuenta de GitHub haciendo click en [este link](https://github.com/apps/r-universe/installations/new) y seleccionando tu cuenta de la primera pantalla que aparece (paso 1 en la siguiente figura). 
+En la segunda pantalla se recomienda seleccionar *all the repositories (todos los repositorios)* (paso 2) y luego hacer click en el botón *Next (siguiente)* (paso 3).
+
+{{< figure src = "r_univserse_instalar_app.png" alt = "ADD ALT TEXT" >}}
+
+Se nos presentará la siguiente imágen dandonos la bienvenida al *universo-r*
+
+{{< figure src = "r_universe_listo.png" alt = "ADD ALT TEXT" >}}
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:92]:
```suggestion
{{< figure src = "r_universe_listo.png" alt = "pantalla de bienvenida al r-universe.  Presenta un mení al lado izquierdo con las opciones (en inglés) de paquetes, artículos, universos, personas que mantienen, setup, acerca de, fuente y ayuda. Al lado derecho, texto de bienvenida e instrucciones sobre r-universe">}}
```

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+
+### Paso 4: R-universe listo
+
+Una vez que se haya instalado la aplicación, el sistema creará automáticamente tu repositorio personal bajo la organización *r-universe*: `https://github.com/r-universe/<tu_nombre_de_usuario>` (ver imagen siguiente). Aquí es donde el sistema mantiene el historial completo de tus paquetes .
+
+{{< figure src = "r_universe_repo_creado.png" alt = "ADD ALT TEXT" >}}
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:99]:
```suggestion
{{< figure src = "r_universe_repo_creado.png" alt = "mi repositorio r-universe" >}}
```

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Después de un par de minutos, por lo general, no más de una hora (en mi caso fueron unos 15 minutos), los paquetes y artículos de los que se hayan completado la compilación en todas las plataformas comenzarán a aparecer en tu panel personal y estarán disponibles para que los usuarios los instalen.
+
+El panel personal se parece a la siguiente figura.
+
+{{< figure src = "r_universe_panel.png" alt = "ADD ALT TEXT" >}}
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:105]:
```suggestion
{{< figure src = "r_universe_panel.png" alt = "El panel personal tiene 3 columnas, el menú de la izquierda es el mismo que en la bienvenida, al medio el listado de paquetes y a la derecha mi perfil sacado de github.  Debajo de mi perfil un listado de las otras personas que mantienen los paquetes que tengo en mi r-universe." >}}
```

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+`![r-universe](https://yabellini.r-universe.dev/badges/<nombre_del_paquete>)`
+
+por ejemplo,
+
+`![r-universe](https://yabellini.r-universe.dev/badges/learnres)`
+
+genera la etiqueta que se ve en la siguiente figura, mostrando la versión del paquete disponible en *r-universe*:
+
+{{< figure src = "r_universe_readme.png" alt = "ADD ALT TEXT" >}}
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:136]:
```suggestion
{{< figure src = "r_universe_readme.png" alt = "readme del paquete learners con la etiqueta de r-universe" >}}
```

**Review by yabellini — COMMENTED** [review]
The change from d to c in Paso 2 is correct.  Thank you.

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+a. Crea un nuevo repositorio llamado `universe` en tu cuenta de GitHub. Para eso ingresas a tu cuenta de github (paso 1),
+
+b. luego haces click en el signo más (+) que se encuentra arriba a la derecha y seleccionas la opción *New repository (nuevo repositorio)* (paso 2).
+
+c. En la pantalla *Create new repository* completamos el *repository name (nombre del repositorio)* con **universe** (paso 3), seleccionamos el repositorio como *public (público)* (paso 4) y lo generamos presionando el botón *Create repository (crear repositorio)* (paso 5).
+
+### Paso 2. detallar los paquetes que agregaremos a nuestro universo
+Una vez que el repositorio se creó se nos mostrará una pantalla con opciones para iniciar ese repo (ver figura siguiente).
+
+{{< figure src = "r_univserse_repo_vacio.png" alt = "ADD ALT TEXT" >}}
```
Comment (steffilazerte) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:67]:
```suggestion
{{< figure src = "r_univserse_repo_vacio.png" alt = "página de github con las opciones: Quick setup - si ya hiciste esto antes, creá un nuevo repositorio desde la línea de comando ó push un repositorio existente desde la linea de comando. La opción creatin a new file está resaltada." >}}
```

**Comment by yabellini** [comment]
@stefaniebutland do you can add my ORCID? https://orcid.org/0000-0002-4522-7466 to the profile?

**Comment by steffilazerte** [comment]
Okay, I got the missing figure, added the alt, the text tweaks, and your orcid. Only thing left is Alt Text for the new figure! Thanks @yabellini!

**Comment by steffilazerte** [comment]
Oops, @yabellini I found two more things I'll need assistance on. 

In the YAML there are two metadata fields for `description` and `tweet`, could you fill those out in Spanish? Thanks!

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  - Yanina Bellini Saibene
+date: '2021-09-21'
+slug: creando-tu-r-universe
+categories: []
+tags:
+  - r-universe
+  - Spanish
+  - community
+package_version: 0.1.0
+description: A very short summary of your post (~ 100 characters)
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:13]:
```suggestion
description: En este post te explico como crear tu r-universe, a partir de la experiencia de crear el mio.
```

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+tweet: A post about blabla by @username!
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:14]:
```suggestion
tweet: En este post @yabellini te explica como crear tu r-universe usando como ejemplo el paso a paso de crear el suyo.
```

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+  - en el campo `package` va el nombre del paquete **tal cual está escrito** en el archivo `DESCRIPTION` que se encuentra en el repo indicado por la url.
+  - Si el paquete R no se encuentra en la raíz del repositorio, también se debe establecer el campo `subdir` en la ruta del directorio raíz del paquete R.
+
+En la figura se ven los paquetes que yo agregué a mi universo.
+
+{{< figure src = "r_univserse_packages.png" alt = "JSON con los paquetes agregados y el formato explicado en el párrafo anterior." >}}
+
+c. Cuando terminamos de completar los datos, presionamos en el botón _Commit new file_. Se mostrará una pantalla similar a la siguiente figura.
+
+{{< figure src = "r_univserse_repo_listo.png" alt = "ALT TEXT HERE" >}}
```
Comment (yabellini) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:81]:
```suggestion
{{< figure src = "r_univserse_repo_listo.png" alt = "repo con el archivo packages.json" >}}
```

**Review by yabellini — COMMENTED** [review]
Add the alt text and the info on the YAML.

**Comment by steffilazerte** [comment]
Great! We're good to publish!

@yabellini I'm going to tweet in English and Spanish on Tuesday when the post goes live, I've included the draft tweets below, let me know if you have any suggestions (especially for the Spanish one, as I used an online translator!)

---

[blog] Creando tu r-universe 🪐⭐

Check out @yabellini's tutorial in Spanish on how to set up your own R-universe 🌌

https://ropensci.org/blog/2021/09/21/creando-tu-r-universe/

Cross-posted from @yabellini's blog https://yabellini.netlify.app/es/post/r_universe/

#RStats

---

[blog] Creando tu r-universe 🪐⭐ 

Echa un vistazo al tutorial de @yabellini en español sobre cómo configurar tu propio universo R 🌌

https://ropensci.org/blog/2021/09/21/creando-tu-r-universe/ 

Publicado en el blog de @yabellini https://yabellini.netlify.app/es/post/r_universe/ 

#RStats

**Comment by yabellini** [comment]
The text is great in both languages.  Thank you!

File: content/blog/2021-09-21-creando-tu-r-universe/index.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
+---
+title: Creando Tu r-Universe
```
Comment (steffilazerte) [inline content/blog/2021-09-21-creando-tu-r-universe/index.md:2]:
```suggestion
title: Creando Tu R-Universe
```

**Pull request merged**